### PR TITLE
Update for CP 1.1.0 compatibility

### LINKED ISSUE #150: pod package needs compatibility update for CP 1.1.0
### Command

```
/usr/local/bin/pod package --dynamic BCFoundation.podspec
```

#### What did you do?

Run the above command - that's all

#### What did you expect to happen?

- It should create a `BCFoundation.framework` containing a dynamic framework
- It should create a podspec to be used with that framework

#### What happened instead?

- It crashes with the below stack trace, not creating any output at all on the file system

### Stack

```
   CocoaPods : 1.1.0.beta.1
        Ruby : ruby 2.0.0p648 (2015-12-16 revision 53162) [universal.x86_64-darwin15]
    RubyGems : 2.0.14.1
        Host : Mac OS X 10.11.5 (15F34)
       Xcode : 7.3.1 (7D1014)
         Git : git version 2.7.4 (Apple Git-66)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ cf8802cc2eb2edce53a2f5839de7b038185e1bb2
```

### Plugins

```
cocoapods-deintegrate : 1.0.0
cocoapods-packager    : 1.4.0
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.0.0
cocoapods-try         : 1.1.0
```

### Error

```
NameError - uninitialized constant Pod::Installer::FileReferencesInstaller
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.1.0.beta.1/lib/cocoapods/sources_manager.rb:170:in `const_missing'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-packager-1.4.0/lib/cocoapods-packager/pod_utils.rb:209:in `install_file_references'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-packager-1.4.0/lib/cocoapods-packager/pod_utils.rb:146:in `install_dynamic_pod'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-packager-1.4.0/lib/pod/command/package.rb:80:in `build_in_sandbox'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-packager-1.4.0/lib/pod/command/package.rb:97:in `block in build_package'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-packager-1.4.0/lib/pod/command/package.rb:96:in `each'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-packager-1.4.0/lib/pod/command/package.rb:96:in `build_package'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-packager-1.4.0/lib/pod/command/package.rb:63:in `run'
/Library/Ruby/Gems/2.0.0/gems/claide-1.0.0/lib/claide/command.rb:334:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.1.0.beta.1/lib/cocoapods/command.rb:50:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.1.0.beta.1/bin/pod:55:in `<top (required)>'
/usr/local/bin/pod:23:in `load'
/usr/local/bin/pod:23:in `<main>'
```

### Podspec (anonymized URLs)

```
Pod::Spec.new do |s|
  s.name             = "BCFoundation"
  s.version          = "0.0.1"
  s.summary          = "BCFoundation contains additions on the level of the Foundation and CoreFoundation frameworks."

  s.homepage         = "https://github.com/<redacted>"
  s.license          = 'Proprietary'
  s.author           = { "Michael Ochs" => "<redacted>" }
  s.source           = { :git => "<redacted>/BCComponents.git", :tag => "foundation-#{s.version.to_s}" }
  
  s.ios.deployment_target = '8.0'

  s.source_files = 'BCFoundation/Classes/**/*'
  s.private_header_files = 'BCFoundation/Classes/**/*+Private.h', 'BCFoundation/Classes/**/_*.h'
  s.frameworks = 'Foundation'
end
```

### PR DESCRIPTION
**Pull request by neonichu**
Fixes #150

- [ ] Update `.gemspec`
- [ ] Ship once CP 1.1.0 is final

### CODE REVIEW

**Comment by ilyawiz** [comment]
can you please release this as a pre-release, so we can use it with cp 1.1 RC /xcode 8 ?

**Comment by danielbowden** [comment]
@ilyawiz rather than wait, in the meantime you can add the following to your `Gemfile`.

```sh
gem 'cocoapods-packager', :git => 'https://github.com/CocoaPods/cocoapods-packager', :branch => 'cp-1.1.0-compatibility'
```

**Comment by jtreanor** [comment]
Would it be possible to get this merged/released now that CocoaPods 1.1 is released?

File: Gemfile [hunk 1]
```diff
@@ -9,4 +9,6 @@ group :development do
   gem 'prettybacon'
   gem 'coveralls', :require => false
   gem 'rubocop'
+
+  gem 'cocoapods', '1.1.0.beta.1'
```
Comment (segiddins) [inline Gemfile]: Please remove this line and run `bundle update cocoapods`

**Comment by segiddins** [comment]
Pushed as 1.5.0

**Pull request merged**